src/commands/synthetics/run-test.ts

```diff
@@ -174,13 +175,19 @@ export class RunTestCommand extends Command {
     return `${host}/${apiPath}`
   }
 
-  private async getTestsToTrigger(api: APIHelper) {
+  private async getTestsList(api: APIHelper) {
     if (this.testSearchQuery) {
       const testSearchResults = await api.searchTests(this.testSearchQuery)
 
       return testSearchResults.tests.map((test) => ({config: this.config.global, id: test.public_id}))
     }
-    const suites = (await getSuites(this.config.files, this.context.stdout.write.bind(this.context.stdout)))
+
+    const files = this.files || [this.config.files]
```

Comment:
The `files` variable actually is a list of globs (the naming in the config may be confusing in the first place), could we rename it to `fileGlobs` so it is more explicit it is a list of globs?

Reply:
I'll rename the variable internally.
But I'll keep the one exposed as `files`, to keep the consistency with the configuration file.